core/src/main/scala/com/codacy/analysis/core/tools/Tool.scala

```diff
@@ -166,12 +166,14 @@ class ToolCollector(toolRepository: ToolRepository) {
       Left(AnalyserError.NoActiveToolInConfiguration)
     } else {
       val toolsIdentified = toolUuids.flatMap { toolUuid =>
-        from(toolUuid, languages).fold(
-          { _ =>
+        from(toolUuid, languages) match {
+          case Left(_: AnalyserError.StandaloneToolInput) =>
+            Set.empty[Tool]
```

Comment:
Not being able to get standalone tools while listing is normal, so we don't need to log anything.

Reply:
Not needed anymore